bin/develop

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+trap "kill 0" EXIT
+trap "exit" INT TERM ERR
+
+if [[ $(type -t subl) == "file" ]]; then 
+  subl ../labs-factfinder-api
+  subl .
+elif [[ $(type -t atom) == "file" ]]; then 
+  atom ../labs-factfinder-api
+  atom .
+elif [[ $(type -t vim) == "file" ]]; then
+  vim ../labs-factfinder-api
+  vim .
+fi
+
+npm run devstart --prefix ../labs-factfinder-api &
+ember s --environment=devlocal &
+
+sleep 10
```

Comment:
if you want to go for full "correctness", try curling localhost:4200 in a loop until you get an expected OK response back instead of just sleeping 10. I feel like there's def examples of that on the internet, it's a pretty standard pattern  in scripts while waiting for services to come up

Reply:
Will do!